shared/fs/path-item-icon.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import * as React from 'react'
+import * as Types from '../constants/types/fs'
+import {globalMargins} from '../styles'
+import {Avatar, Icon} from '../common-adapters'
+
+type PathItemIconProps = {
+  spec: Types.PathItemIconSpec,
+}
+
+const PathItemIcon = ({spec}: PathItemIconProps) => {
+  switch (spec.type) {
+    case 'teamAvatar':
+      return <Avatar size={32} teamname={spec.teamName} isTeam={true} style={iconStyle} />
```

Comment:
These sizes will need to be different on mobile, won't they? Did you test this on mobile?

Reply:
Good point. I haven't tested this on mobile yet; was planning to do that altogether in my popup menu ticket.